config/development.json

```diff
@@ -42,7 +42,6 @@
 		"code-splitting": true,
 		"comments/filters-in-posts": true,
 		"comments/moderation-tools-in-posts": true,
-		"comments/management/comment-view": true,
```

Comment:
Just a note that after https://github.com/Automattic/wp-calypso/pull/20972 lands we'll need to perform this removal in other environments too.

Reply:
😉 updated on ec43f156f447b40af6a9dcec67b4469c95de2c1c.